pbsmetrics/prometheus/prometheus.go

```diff
@@ -294,9 +295,9 @@ func (me *Metrics) RecordUserIDSet(userLabels pbsmetrics.UserLabels) {
 
 func resolveLabels(labels pbsmetrics.Labels) prometheus.Labels {
 	return prometheus.Labels{
-		demandSourceLabel: string(labels.Source),
-		requestTypeLabel:  string(labels.RType),
-		// "pubid":   labels.PubID,
+		demandSourceLabel:   string(labels.Source),
+		requestTypeLabel:    string(labels.RType),
+		accountLabel:        string(labels.PubID),
```

Comment:
There other 3 instances of the commented `// "pubid":   labels.PubID,` line in functions `resolveAdapterLabels()`, `resolveBidLabels()`, and `resolveAdapterErrorLabels()`. Shouild we also add `accountLabel` there? After all, `demandSourceLabel`, `requestTypeLabel`, `browserLabel`, `cookieLabel`, and `responseStatusLabel` do get repeated in all of them.

Reply:
Discussed in a meeting. LGMT